src/utils/chart.ts

```diff
@@ -170,21 +171,24 @@ const updateLinksAndPorts = (
     properties: { nodeId: toNode.id },
   };
 
+  const properties: LinkProperties = {
```

Comment:
Just moved this into its own constant so I can specify the type and added the `isPrivate` field.